[2024-11-28T06:55:09.119+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T06:55:09.163+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_green_tripdata.check_load_tripdata scheduled__2024-11-27T00:00:00+00:00 [queued]>
[2024-11-28T06:55:09.184+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_green_tripdata.check_load_tripdata scheduled__2024-11-27T00:00:00+00:00 [queued]>
[2024-11-28T06:55:09.185+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-28T06:55:09.220+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_load_tripdata> on 2024-11-27 00:00:00+00:00
[2024-11-28T06:55:09.235+0000] {standard_task_runner.py:64} INFO - Started process 771 to run task
[2024-11-28T06:55:09.241+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nyc_green_tripdata', 'check_load_tripdata', 'scheduled__2024-11-27T00:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/dag_green_tripdata.py', '--cfg-path', '/tmp/tmpriqw65l9']
[2024-11-28T06:55:09.247+0000] {standard_task_runner.py:91} INFO - Job 88: Subtask check_load_tripdata
[2024-11-28T06:55:09.338+0000] {task_command.py:426} INFO - Running <TaskInstance: nyc_green_tripdata.check_load_tripdata scheduled__2024-11-27T00:00:00+00:00 [running]> on host 39c047f38cb8
[2024-11-28T06:55:09.506+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='nyc_green_tripdata' AIRFLOW_CTX_TASK_ID='check_load_tripdata' AIRFLOW_CTX_EXECUTION_DATE='2024-11-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-27T00:00:00+00:00'
[2024-11-28T06:55:09.510+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T06:55:10.539+0000] {config_helper.py:85} INFO - Trying to create config YAML file ~/.soda/config.yml ...
[2024-11-28T06:55:10.545+0000] {soda_telemetry.py:61} INFO - Setting up usage telemetry.
[2024-11-28T06:55:10.634+0000] {logging_mixin.py:188} INFO - Running Soda Scan ...
[2024-11-28T06:55:10.644+0000] {log.py:101} INFO - [06:55:10] Soda Core 3.4.1
[2024-11-28T06:55:12.813+0000] {log.py:101} INFO - [06:55:12] Using application default credentials.
[2024-11-28T06:55:14.629+0000] {log.py:101} INFO - [06:55:14] Scan summary:
[2024-11-28T06:55:14.630+0000] {log.py:101} INFO - [06:55:14] 1/1 check PASSED: 
[2024-11-28T06:55:14.631+0000] {log.py:101} INFO - [06:55:14]     green_tripdata in nyc_green_taxi
[2024-11-28T06:55:14.633+0000] {log.py:101} INFO - [06:55:14]       Schema Check [include/soda/checks/sources/green_tripdata.yml] [PASSED]
[2024-11-28T06:55:14.634+0000] {log.py:101} INFO - [06:55:14]         schema_measured = [vendor_id INT64, lpep_pickup_datetime TIMESTAMP, lpep_dropoff_datetime TIMESTAMP, store_and_fwd_flag STRING, rate_code_id INT64, pu_location_id INT64, do_location_id INT64, passenger_count INT64, trip_distance FLOAT64, fare_amount FLOAT64, extra FLOAT64, mta_tax FLOAT64, tip_amount FLOAT64, tolls_amount FLOAT64, ehail_fee INT64, improvement_surcharge FLOAT64, total_amount FLOAT64, payment_type INT64, trip_type INT64, congestion_surcharge FLOAT64, __index_level_0__ INT64]
[2024-11-28T06:55:14.635+0000] {log.py:101} INFO - [06:55:14] All is good. No failures. No warnings. No errors.
[2024-11-28T06:55:14.636+0000] {log.py:101} INFO - [06:55:14] Sending results to Soda Cloud
[2024-11-28T06:55:15.755+0000] {log.py:101} INFO - [06:55:15] Soda Cloud Trace: 4025034285267344861
[2024-11-28T06:55:15.777+0000] {logging_mixin.py:188} INFO - INFO   | Soda Core 3.4.1
DEBUG  | Reading configuration file "include/soda/configuration.yml"
DEBUG  | Reading SodaCL file "include/soda/checks/sources/green_tripdata.yml"
DEBUG  | Scan execution starts
INFO   | Using application default credentials.
DEBUG  | Query 1.nyc_green_taxi.green_tripdata.schema[green_tripdata]:
SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'green_tripdata';
INFO   | Scan summary:
DEBUG  | 1/1 query OK
DEBUG  |   1.nyc_green_taxi.green_tripdata.schema[green_tripdata] [OK] 0:00:01.563738
INFO   | 1/1 check PASSED: 
INFO   |     green_tripdata in nyc_green_taxi
INFO   |       Schema Check [include/soda/checks/sources/green_tripdata.yml] [PASSED]
INFO   |         schema_measured = [vendor_id INT64, lpep_pickup_datetime TIMESTAMP, lpep_dropoff_datetime TIMESTAMP, store_and_fwd_flag STRING, rate_code_id INT64, pu_location_id INT64, do_location_id INT64, passenger_count INT64, trip_distance FLOAT64, fare_amount FLOAT64, extra FLOAT64, mta_tax FLOAT64, tip_amount FLOAT64, tolls_amount FLOAT64, ehail_fee INT64, improvement_surcharge FLOAT64, total_amount FLOAT64, payment_type INT64, trip_type INT64, congestion_surcharge FLOAT64, __index_level_0__ INT64]
INFO   | All is good. No failures. No warnings. No errors.
INFO   | Sending results to Soda Cloud
INFO   | Soda Cloud Trace: 4025034285267344861
[2024-11-28T06:55:15.778+0000] {dag_green_tripdata.py:142} INFO - Result from check: 0 of type <class 'int'>
[2024-11-28T06:55:15.779+0000] {python.py:237} INFO - Done. Returned value was: 0
[2024-11-28T06:55:15.781+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T06:55:15.833+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nyc_green_tripdata, task_id=check_load_tripdata, run_id=scheduled__2024-11-27T00:00:00+00:00, execution_date=20241127T000000, start_date=20241128T065509, end_date=20241128T065515
[2024-11-28T06:55:15.910+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-28T06:55:15.983+0000] {taskinstance.py:3503} INFO - 6 downstream tasks scheduled from follow-on schedule check
[2024-11-28T06:55:15.992+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
